src/hardware/sblaster.h

```diff
@@ -32,6 +32,10 @@ class SBLASTER final {
 
 	void SetChannelRateHz(const int requested_rate_hz);
 
+	// Returns true of the channel was sleeping and was awoken.
+	// Returns false if the channel was already awake.
+	bool MaybeWakeup();
```

Comment:
It's "wake up", so this should be called `MaybeWakeUp()`.

Reply:
Fixed; thanks.